lightning/src/routing/scoring.rs

```diff
@@ -1177,10 +1177,20 @@ impl<T: Time> Readable for ChannelLiquidity<T> {
 			(2, max_liquidity_offset_msat, required),
 			(4, duration_since_epoch, required),
 		});
+		// On rust prior to 1.60 `Instant::duration_since` will panic if time goes backwards.
+		// Because we calculate "now" against wallclock time when we were written are reload it
+		// here, we may cause time to go backwards if wallclock time is before when we were
+		// written. Thus, we check if we'd end up with a time in the future and juse use `now`
+		// instead to avoid panicing later.
```

Comment:
what if there's some actual bug and we end up hiding it because of this? (if diff is alot and weird)
shouldn't the solution be to use ```saturating_duration_since``` for now ?
(that will also mean that we will clean it up when duration since is not panicking)

Reply:
main intention was for cleanup, but i guess it could be safer to do it at one place at read time.
first question was about if we should do a tolerance check on difference.